zuul/tenants.yaml

```diff
@@ -94,6 +94,7 @@
               - ansible/galaxy
               - ansible/pylibssh
               - ansible/workshops
+              - project-receptor/receptor
```

Comment:
Are you planing to just do 3pci or gate receptor with zuul?

Also, we need to make sure https://github.com/apps/ansible-zuul is installed on the repo in github

Reply:
In the first time yes, I am planning to do 3pci with Zuul. This will be revisited once current developer will see it in actions for a moment.

I have already pinged @matburt for the `apps/ansible-zuul` this AM - will be done shortly